Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -526,6 +526,31 @@ InitializeSmbiosInfo (
   //
   // SMBIOS_TYPE_PROCESSOR_INFORMATION : TBD
   //
+  //
+  // SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_PHYSICAL_MEMORY_ARRAY,
+    1, "2");
+
+  //
+  // SMBIOS_TYPE_MEMORY_DEVICE
+  //
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    1, "Controller0-ChannelA");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    2, "BANK 0");
+  AddSmbiosTypeString (&TempSmbiosStrTbl[Index++], SMBIOS_TYPE_MEMORY_DEVICE,
+    3, "Micron Technology");
```

Comment:
Not sure if we could detect/get the manufacturer info from somewhere, if no we could use the configuration data I think.

Reply:
It is possible to get manufacturer info from SPD (easier if using hardcoded memory SPD, but it will need more change for smbus). For DDR4/LPDDR4, the manufacturer info locates at offset 0x140-141.